conda.recipe/meta.yaml

```diff
@@ -0,0 +1,105 @@
+{% set data = load_setup_py_data() %}
+
+package:
+  name: tethysplatform
+  version: {{ data['version'] }}
+
+
+source:
+  path: ..
+
+build:
+  number: 0
+  noarch: python
+#  script:  python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python >=3.6
+
+    # system dependencies
+    - pycrypto
```

Comment:
Have we made a decision on whether these should be locked down or not? Perhaps this file should be generated with options to lock all dependencies for release builds, or leave it open for dev builds?

Reply:
This is an interesting idea. Depending on if we want to release a PyPI package then we'll need to maintain several versions of the dependencies. Having them in a template might be a good way to consolidate that.